src/main/java/de/themorpheus/edu/taskservice/controller/TaskController.java

```diff
@@ -115,4 +118,20 @@ public void deleteTask(int taskId) {
 
 		return ControllerResult.of(this.taskRepository.save(taskModel));
 	}
+
+	public ControllerResult<Object> voteTask(int taskId, int vote) {
```

Comment:
require the userId and generate the "fake" userId in the endpoint, so we only have to change the endpoint later